docs/CreateKoblenzHash.md

```diff
@@ -15,14 +14,11 @@ The example data is
 ```agsl
 //Example:
 {
-    full_name: "Karla Koblenz"
     birthday: 12213
     referenceNumber: "123K"
 }
 ```
 
-- The full_name must be `FirstnameSpaceLastname`. Every char must exactly match the user input, as otherwise there is not possibility to match the data Koblenz transfers with the input the user makes.
-e.g. `Karla Koblenz` will match neither with `Karla Lisa Koblenz` nor with `Karlá Koblenz`.
 - The birthday is defined in our protobuf [card.proto](../frontend/card.proto) file: It counts the days since the birthday (calculated from 1970-01-01).
   All values of this field are valid, including the 0, which indicates that the birthday is on 1970-01-01. Birthdays before 1970-01-01 have negative values.
 - referenceNumber is set to the "Aktenzeichen"
```

Comment:
to be added:
> "The file numbers (Aktenzeichen) currently have a minimum length of five characters and a maximum length of 13 characters, but we ask that the solution be technically prepared for 15 characters." - written by Koblenz today

Reply:
i think that should be fine. We have to limit this in the frontend. We had some kind of signature buffer which will not survive :(
But even we had the same issue with the extension of the card holder name to 40 chars